src/modules/PopupMovies.js

```diff
@@ -100,7 +98,15 @@ class PopupMovies {
       return response.ok;
     };
 
+    countComments = async () => {
+      const movieComments = await this.getComment();
+      const count = document.querySelector('.count');
+      const numberOf = movieComments.length;
+      count.innerText = numberOf;
+    }
```

Comment:
- [ ] Please consider implement the `countComments` as a separate module as written in the [linked issue](https://github.com/Trast00/Capston-javascript/issues/5#issue-1516502944)

Reply:
alright sir, i will do that in the next pull request